x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -17,11 +17,13 @@ import React from 'react';
 import { generatePath } from 'react-router-dom';
 import { pagePathGetters } from '@kbn/fleet-plugin/public';
 import { i18n } from '@kbn/i18n';
+import { PackagePolicy } from '@kbn/fleet-plugin/common';
```

Comment:
```suggestion
import type { PackagePolicy } from '@kbn/fleet-plugin/common';
```

this one might actually be important, not sure but might end up requiring fleet-plugin common